### ISSUE TITLE
cannot create new user

### ISSUE THREAD

**Issue by martinjrobins** [Battery-Intelligence-Lab/galvanalyser#48]
**Describe the bug**

When registering new user in login, the UI seems to create the user, but the api call is rejected

```
APIConnection.ts:113          POST http://localhost:5000/users/ 405 (Method Not Allowed)
```

**To Reproduce**

Try to register a new user on the login page

**Comment by martinjrobins** [comment]
perhaps because the viewset is read-only?

```python
class UserViewSet(viewsets.ReadOnlyModelViewSet):
```

**Comment by mjaquiery** [comment]
Closed by #50
